render_delegate/node_graph.cpp

```diff
@@ -814,11 +812,8 @@ HdArnoldNodeGraph::NodeDataPtr HdArnoldNodeGraph::GetNode(const SdfPath& path, c
 
 AtNode *HdArnoldNodeGraph::GetMaterialxShader(const AtString &nodeType, const AtString &nodeName, AtParamValueMap *params)
 {
-    // Disable materialx support until it's included in the USD libs
-    // that are shipped with Arnold
-#ifdef ARNOLD_MATERIALX
-    
-//#if ARNOLD_VERSION_NUM < 70103
+    // MaterialX support in USD was added in Arnold 7.1.4
+#if ARNOLD_VERSION_NUM < 70104
```

Comment:
should that be superior or equal to 70104 ?

Reply:
oups

Reply:
fixed